app/controllers/MathSwipeController.coffee

```diff
@@ -124,14 +125,18 @@ class MathSwipeController
   generateBoard: (inputs, length, solutionPlacements) ->
     DFS.setEquationsOnGrid length, inputs, AdjacentCellsCalculator, solutionPlacements
 
-  generateInputs: (inputLengths, inputs, goals) ->
+  generateInputs: (inputLengths, goals = [], inputs = []) ->
```

Comment:
I'd rather have ```goals = [], inputs = []``` as the first two lines of the method since you're only calling @generateInputs using the inputLengths